ikarus/utils/tensorutils.hh

```diff
@@ -322,4 +322,77 @@ auto fromVoigt(const Eigen::Matrix<ScalarType, 6, 6>& CVoigt) {
   return C;
 }
 
+/**
+ * @brief Calculates the 2D transformation matrix for Enhanced Assumed Strains (EAS).
+ *
+ * This function computes the transformation matrix for 2D EAS using the provided geometry.
+ *
+ * @tparam Geometry The geometry type.
+ * @param geometry Reference to the geometry object.
+ * @return Transformation matrix for 2D EAS.
```

Comment:
Now this method is not related to EAS and not only an implementation details. Write the correct comment here. Also provide an example code: \code ... \endcode what this method is actually doing. Same for 3D.

Reply:
Also provide the local coordinate as an argument

Reply:
Yes. This function is very hard-coded. As I have mentioned in the PR description, I will refactor these functions completely while handling #248. I will also add detailed doxygen comments then and update documentation with more description. For this PR, I am just removing the reference to EAS.